src/frontend/mame/luaengine.cpp

```diff
@@ -1283,6 +1283,18 @@ void lua_engine::initialize()
  * driver.manufacturer
  * driver.compatible_with
  * driver.default_layout
+ * driver.orientation - screen rotation degree (rot0/90/180/270)
```

Comment:
This isn't necessarily going to give you a meaningful result.  Individual screens can also have rotation flags, and some systems have multiple screens with different orientations.

Reply:
You're right, it looks like a legacy flag obsoleted by `screen_device::orientation()`. Should I remove it? Seems to be easier to commit straight to master for whoever has access than setting up a new PR (unless my updates require such a clean-up PR).